admin/static/js/spam/spamList.js

```diff
@@ -0,0 +1,30 @@
+/** Facilitate actions in the list view of spam.
```

Comment:
I will admit to not being up on my knockout but I don't think this does anything. :)